mlflow/pyfunc/__init__.py

```diff
@@ -2092,11 +2092,11 @@ def spark_udf(
 
     if use_dbconnect_artifact:
         udf_sandbox_info = get_dbconnect_udf_sandbox_info(spark)
-        if Version(udf_sandbox_info.mlflow_version) < Version("2.19.0"):
+        if Version(udf_sandbox_info.mlflow_version) < Version("2.18.0"):
```

Comment:
did you manually test this?

Reply:
writing test for this is tricky, and the logic is quite simple, so I think no test is ok.